shared/engine/index.js

```diff
@@ -1,23 +1,49 @@
+// @flow
 // Handles sending requests to native mobile (then go) and back
-
-import engine from './index.native'
+import NativeEventEmitter from '../common-adapters/native-event-emitter'
 import Transport from './transport'
+import nativeEngine from './index.native'
 import rpc from 'framed-msgpack-rpc'
-import {printRPC, printOutstandingRPCs} from '../local-debug'
 import setupLocalLogs from '../util/local-log'
 import {Buffer} from 'buffer'
-import NativeEventEmitter from '../common-adapters/native-event-emitter'
-import {log} from '../native/log/logui'
 import {constants} from '../constants/types/keybase-v1'
+import {log} from '../native/log/logui'
+import {printRPC, printOutstandingRPCs} from '../local-debug'
+import type {incomingCallMapType} from '../constants/types/flow-types'
 
 const {client: {Client: RpcClient}} = rpc
+const KEYBASE_RPC_DELAY_RESULT: number = process.env.KEYBASE_RPC_DELAY_RESULT ? parseInt(process.env.KEYBASE_RPC_DELAY_RESULT) : 0
+const KEYBASE_RPC_DELAY: number = process.env.KEYBASE_RPC_DELAY ? parseInt(process.env.KEYBASE_RPC_DELAY) : 0
 
 const NO_ENGINE = process.env.KEYBASE_NO_ENGINE
 if (NO_ENGINE) {
   console.log('Engine disabled!')
 }
 
+type SessionID = number;
+type SessionIDKey = string; // used in our maps, really converted to a string key
```

Comment:
map keys have to be strings and not numbers to flow wants us to be explicit about this. instead of just typing string i made this alias cause the maps we have are sometimes confusing so this makes it more clear what the key actually is semantically